models/ed/README.md

```diff
@@ -3,6 +3,7 @@
 <!-- badges: start -->
 
 [![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
+[![PEcAn.ED2 status badge](https://pecanproject.r-universe.dev/badges/PEcAn.ED2)](https://pecanproject.r-universe.dev)
```

Comment:
You can also have the badge link to the specific page for the package. It's up to you.
```suggestion
[![PEcAn.ED2 status badge](https://pecanproject.r-universe.dev/badges/PEcAn.ED2)](https://pecanproject.r-universe.dev/PEcAn.ED2)
```

Reply:
Yes even I was thinking of doing that. Made the changes : )